social-distancing/libs/ConfigEngine.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
```

Comment:
This looks nice. Thanks for providing the config engine. Can you please add documentation and example use cases for this?